test/NuGet.Server.Core.Tests/ServerPackageRepositoryTest.cs

```diff
@@ -387,7 +416,12 @@ public void ServerPackageRepositoryEmptyRepo()
                 var findPackagesById = serverRepository.FindPackagesById("test");
                 var getPackages = serverRepository.GetPackages().ToList();
                 var getPackagesWithDerivedData = serverRepository.GetPackages().ToList();
-                var getUpdates = serverRepository.GetUpdates(Enumerable.Empty<IPackageName>(), true, true, Enumerable.Empty<FrameworkName>(), Enumerable.Empty<IVersionSpec>());
+                var getUpdates = serverRepository.GetUpdates(
+                    Enumerable.Empty<IPackageName>(),
+                    true,
+                    true,
```

Comment:
Use either named parameters or inline comments (e.g.:  /*includePrerelease*/) to explain these mysterious literal arguments.